Makefile

```diff
@@ -0,0 +1,46 @@
+include project.mk
```

Comment:
Please use the standard format used by all other Makefiles: See https://github.com/lisa/configure-alertmanager-operator/blob/a4483abaffc9fb0acedf07992897294ca1a1297c/Makefile for reference. Customizations can be made after the mark.

Reply:
Would it help to have a PR into this fork to apply the standard bits?  Otherwise might suggest we tackle that in a separate PR and not mix it into this initial merge.  It's important, but so is getting a good baseline into `master`.

Reply:
I did, https://github.com/dak1n1/configure-alertmanager-operator/pull/3, but it was closed.

Reply:
I'd prefer to get this initial commit in before having other contributors jump in. I tried merging Lisa's changes into my branch, but with all the merge conflicts, it was hard to get the changes in and hard to know which parts were broken. I had added them to this PR, but I had to revert the changes to the Docker file because I couldn't get it to build. It's too many moving parts. I'll just get this PR merged ASAP for a baseline so others can begin contributing to the repo.

Reply:
Added standard make file.